lib/src/package_resolver.dart

```diff
@@ -50,15 +47,12 @@ abstract class PackageResolver {
   ///
   /// Note that if this is a `data:` URI, it may not be safe to pass as a
   /// parameter to a Dart process due to length limits.
-  ///
-  /// Completes to `null` when using a [packageRoot] for resolution, or when no
-  /// package resolution is being used.
   Future<Uri> get packageConfigUri;
 
-  /// The base URL for resolving `package:` URLs.
+  /// **DEPRECATED** Will always return `null`.
```

Comment:
This isn't true. A user can still construct a package resolver by hand that uses a package root to resolve URLs.

Reply:
I guess we could say that the default impl will return `null`...